frontend/src/v5/store/drawings/drawings.selectors.ts

```diff
@@ -32,6 +32,15 @@ import { Calibration } from './drawings.types';
 
```

Comment:
Uploading to a specific drawing does not populate the destination field (although it does show the drawing as being used)


Uploading Registrazione 2024-11-18 112435.mp4…